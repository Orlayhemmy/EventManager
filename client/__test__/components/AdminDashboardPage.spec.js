// import React from 'react';
// import expect from 'expect';
// import AdminDashboardPage
//   from '../../components/AdminDashboard/Container/AdminDashboardPage';


// describe('Admin Component', () => {
//   const props = {
//     location: {
//       pathname: "/admin-dashboard"
//     },
//     auth: {
//       isAuth: true
//     }
//   };
//   it('should render admin dashboard page', () => {
//   const wrapper = shallow(<AdminDashboardPage {...props}/>);
//     expect(wrapper).toMatchSnapshot();
//   });
// });



