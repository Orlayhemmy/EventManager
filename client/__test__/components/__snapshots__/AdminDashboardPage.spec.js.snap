// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin Component should render admin dashboard page 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AdminDashboardPage
    auth={
      Object {
        "isAuth": true,
      }
    }
    location={
      Object {
        "pathname": "/admin-dashboard",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Connect(NavBar)
          path="/admin-dashboard"
        />,
        <Connect(AdminPanelPage)
          path="/admin-dashboard"
        />,
        <Footer />,
      ],
      "id": "center-page",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "path": "/admin-dashboard",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "path": "/admin-dashboard",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Connect(NavBar)
            path="/admin-dashboard"
          />,
          <Connect(AdminPanelPage)
            path="/admin-dashboard"
          />,
          <Footer />,
        ],
        "id": "center-page",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "path": "/admin-dashboard",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "path": "/admin-dashboard",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
